src/components/NetworkAssetSelectInput/NetworkAssetSelectInput.tsx

```diff
@@ -318,6 +329,24 @@ const NetworkAssetSelectInput = ({
     setAssetSearchQuery('');
   }
 
+  const formatBalanceByChainByAccountType = (
+    supportedChain: Chain,
+    accType?: string
+  ) => {
+    if (accType === 'Contract' && label === 'From') {
+      return balancePerChainSmartWallet && balancePerChainSmartWallet.length
+        ? '・$' +
+            balancePerChainSmartWallet?.filter(
+              (item: any) => item.chain === supportedChain.chainId
+            )[0]?.total
+        : '・$0';
+    }
+    else if(accType === 'Key' && label === 'From'){
```

Comment:
please be consistent with coding style
```suggestion
    } else if (accType === 'Key' && label === 'From') {
```